protocol/messenger_sync_activity_center_test.go

```diff
@@ -0,0 +1,121 @@
+package protocol
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+type MessengerSyncActivityCenterSuite struct {
+	MessengerBaseTestSuite
+}
+
+func TestMessengerSyncActivityCenter(t *testing.T) {
+	suite.Run(t, new(MessengerSyncActivityCenterSuite))
+}
+
+func (s *MessengerSyncActivityCenterSuite) TestSyncUnRead() {
+	theirMessenger, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
+	s.NoError(err)
+	PairDevices(&s.Suite, theirMessenger, s.m)
```

Comment:
I know it's no obvious but it should be manually done in both directions.
It won't probably affect the test, but better to fix to catch any loop syncing.
```suggestion
	PairDevices(&s.Suite, theirMessenger, s.m)
	PairDevices(&s.Suite, s.m, theirMessenger)
```